docs/modules/hivemq.md

```diff
@@ -0,0 +1,358 @@
+# HiveMQ Module
+
+# HiveMQ Test Container
```

Comment:
There should only be 1 header with level one

Reply:
That is correct. I wonder if this is also the reason, why the page is not added to the navigation pane:
![image](https://user-images.githubusercontent.com/5088104/148222109-35846930-04e8-45e7-af30-2ba011b72b2b.png)

Reply:
It also needs to be referenced from `nav` in `mkdocs.yml`

Reply:
done